### PR TITLE
Set up customs information for international shipments

### PR DESCRIPTION
**Pull request by acreilly**

### CODE REVIEW

File: app/decorators/models/solidus_easypost/spree/shipment_decorator.rb [hunk 1]
```diff
@@ -30,12 +34,18 @@ def selected_easy_post_shipment_id
       end
 
       def build_easypost_shipment
-        ::EasyPost::Shipment.create(
+        shipment_hash = {
           to_address: order.ship_address.easypost_address,
           from_address: stock_location.easypost_address,
           parcel: to_package.easypost_parcel,
           options: ::Spree::Easypost::Config.ddp_enabled ? { incoterm: 'DDP' } : {}
-        )
```
Comment (dhonig) [inline app/decorators/models/solidus_easypost/spree/shipment_decorator.rb:38]: Whats this?  What does DDP mean in this context?

File: app/models/solidus_easypost/customs_info.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        non_delivery_option: 'abandon',
+        restriction_type: 'none',
+        restriction_comments: ''
+      }
+    end
+
+    private
+
+    def eel_pfc
+      if order.total < 2500
```
Comment (dhonig) [inline app/models/solidus_easypost/customs_info.rb:39]: Why 2500?  why is that amount special?

File: app/models/solidus_easypost/customs_info.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        'NOEEI 30.37(a)'
```
Comment (dhonig) [inline app/models/solidus_easypost/customs_info.rb:40]: This code probably relates to the above question?

File: spec/models/solidus_easypost/customs_info_spec.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe SolidusEasypost::CustomsInfo, :vcr do
```
Comment (dhonig) [inline spec/models/solidus_easypost/customs_info_spec.rb:5]: Consider adding comments that describe the rules being enforced.

File: app/decorators/models/solidus_easypost/spree/shipment_decorator.rb (hunk 1 above)
Comment (acreilly) [reply app/decorators/models/solidus_easypost/spree/shipment_decorator.rb:38]: Addresses the issue here: https://github.com/solidusio-contrib/solidus_easypost/issues/63

File: app/models/solidus_easypost/customs_info.rb (hunk 2 above)
Comment (acreilly) [reply app/models/solidus_easypost/customs_info.rb:39]: eel_pfc = When shipping outside the US, you need to provide either an Exemption and Exclusion Legend (EEL) code or a Proof of Filing Citation (PFC). Which you need is based on the value of the goods being shipped.

- If the value of the goods is less than $2,500, then you pass the following EEL code: "NOEEI 30.37(a)"
- If the value of the goods is greater than $2,500, you need to get an Automated Export System (AES) Internal Transaction Number (ITN) for your shipment. ITN will look like “AES X20120502123456”. To get an ITN, go to the AESDirect website.

An ITN is required for any international shipment valued over $2,500 and/or requires an export license unless exemptions apply.

File: spec/models/solidus_easypost/customs_info_spec.rb (hunk 4 above)
Comment (acreilly) [reply spec/models/solidus_easypost/customs_info_spec.rb:5]: Will do

**Pull request merged**